tests/main/interfaces-time-control/task.yaml

```diff
@@ -5,34 +5,63 @@ details: |
     can access the /dev/rtc device node exposed by a slot on the OS
     snap properly.
 
-systems: [ubuntu-core-16-64]
+systems: [-opensuse-*,-fedora-*,-ubuntu-core-*]
 
 prepare: |
-    if [ "$TRUST_TEST_KEYS" = "false" ]; then
-        echo "This test needs test keys to be trusted"
-        exit
-    fi
+    # Install a snap declaring a plug on time-control
+    cp $TESTSLIB/snaps/test-snapd-timedate-control-consumer/meta/snap.yaml $TESTSLIB/snaps/test-snapd-timedate-control-consumer/meta/snap.yaml.orig
+    sed 's/\[\]/\[time-control\]/g' -i $TESTSLIB/snaps/test-snapd-timedate-control-consumer/meta/snap.yaml
```

Comment:
May I ask why do you need this? Can we just get a snap with appropriate plugs. This is just clouding the test IMO